templates/index.html

```diff
@@ -1,7 +1,124 @@
-<title>Community website</title>
-<body>
-<ul>
-    <li><a href="/gci/">Google Code-in</a>
-    <li><a href="/activity/">GitHub activity</a>
-</ul>
-</body>
+<!DOCTYPE html>
+  <html>
+    <head>
+      <!--Import Google Icon Font-->
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+      <!--Import materialize.css-->
+      <link type="text/css" rel="stylesheet" href="../static/css/materialize.min.css"  media="screen,projection"/>
+        <style>
+        nav {
```

Comment:
Please create a new `style.css` for this don't include CSS in here, we may need to add new things in future.